src/AzureExtension/DevBox/DevBoxProvider.cs

```diff
@@ -159,11 +159,12 @@ public IAsyncOperation<ComputeSystemsResult> GetComputeSystemsAsync(IDeveloperId
                 ArgumentNullException.ThrowIfNull(developerId);
                 ArgumentNullException.ThrowIfNullOrEmpty(developerId.LoginId);
 
-                _log.Information($"Attempting to retrieving all Dev Boxes for {developerId.LoginId}, at {DateTime.Now}");
+                var start = DateTime.Now;
+                _log.Information($"Attempting to retrieving all Dev Boxes for {developerId.LoginId}");
 
                 var computeSystems = await GetDevBoxesAsync(developerId);
 
-                _log.Information($"Successfully retrieved all Dev Boxes for {developerId.LoginId}, at {DateTime.Now}");
+                _log.Information($"Successfully retrieved all Dev Boxes for {developerId.LoginId}, in {DateTime.Now - start}");

```

Comment:
Consider using the [stopwatch class](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch?view=net-8.0) to keep track of how long the task took.

This method is prone to errors via DST.

Reply:
I believe a less error-prone method of getting elapsed time is to use UTCNow.